_posts/2024-10-04-duckdb-user-survey-analysis.md

```diff
@@ -0,0 +1,252 @@
+20% of users processed a dataset between 100 GB and 1 TB and approximately 5% of the users ventured into the 1 TB+ territory. About 1% processed 10 TB+ datasets.
+These findings are in line with [statistics derived from a recent RedShift usage dataset](https://motherduck.com/blog/redshift-files-hunt-for-big-data/#whos-got-big-data) by [Jordan Tigani of MotherDuck](https://motherduck.com/authors/jordan-tigani/), and the recent analysis of the [Snowflake and RedShift datasets](https://www.fivetran.com/blog/how-do-people-use-snowflake-and-redshift) by [George Fraser of Fivetran](https://www.fivetran.com/people/george-fraser).
+
+![Dataset sizes](/images/blog/survey/dataset_sizes.svg)
+
+While these results obviously are somewhat biased – users who need to crunch through huge datasets may not work with DuckDB (yet!) –, the skew towards smaller datasets is quite significant and shows that many real-world use cases can be tackled using small to medium-sized datasets. The results also show that DuckDB *can* solve many problems on datasets larger than 1 TB.
+
+## Features
+
+### Most Liked Features
+
+We were curious: what do users like most about DuckDB? The plot shows the most frequent responses:
+![Most liked DuckDB features](/images/blog/survey/most_liked_features.svg)
+
+The most liked feature, by far, is **high performance**.
+Users also enjoy **file format support** (CSV, Parquet, JSON, etc.),
+**ease of use**,
+**extensive SQL support** (including [friendly SQL]({% link docs/sql/dialect/friendly_sql.md %}))
+and **in-memory integrations** such as support for Pandas, Arrow and NumPy.
+Finally, users mentioned low memory usage, protocol support (e.g., HTTPS, S3), database integrations, and portability.
+
+### Feature Requests
+
+We asked users about the features that they'd most like to see in future DuckDB versions. The most popular requests are listed in the table below:
+
+| Feature                                                                 | Percentage |
+|:------------------------------------------------------------------------|-----------:|
+| Improved partitioning and optimizations related to partitioning         | 39%        |
+| Improved support for time series and optimizations for pre-sorted data  | 35%        |
+| Support for materialized views                                          | 28%        |
+| Support for vector search                                               | 24%        |
+| Support for attaching to database systems via ODBC                      | 24%        |
+| Support for time travel queries (query the database as of a given time) | 23%        |
+| Support for the Delta Lake format                                       | 22%        |
+| Improved support for Iceberg (including writes)                         | 17%        |
+
+We are happy to report that, since a few months have passed since the survey, some of these requests have already been implemented:
```

Comment:
```suggestion
We are happy to report that, since survey was conducted pre-v1.0.0 and DuckDB is now at version 1.1.1, some of these requests are already a reality:
```
Possibly? basically instead of 'month' measuring time as releases.